github.js

```diff
@@ -18,23 +18,26 @@ const GITHUB_AUTH_HEADERS = {
 /**
  * @param {String} filePath - path where config file is stored
  * @param {String} repoName - name of the github repository
+ * @param {boolean} isCustomPath - true if users pass a path flag to the CLI
+ * @param {String | undefined} - branch to fetch from. if undefined, default branch used
  * @returns {Buffer|Null} Zip data buffer
  */
 async function fetchJsonFromRepository(
-  repoName = 'HubSpot/hubspot-project-components',
```

Comment:
This theoretically will necessitate a major release. It wasn't entirely necessary to remove, but I felt that the CLI was relying on the default value far too much which made things very confusing

Reply:
+1 to removing this 👍